packages/turbo-workspace-convert/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@turbo/workspace-convert",
+  "version": "0.0.0",
+  "description": "Quickly switch between workspace managers",
+  "homepage": "https://turborepo.org",
+  "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vercel/turborepo",
+    "directory": "packages/use-workspaces"
+  },
+  "bugs": {
+    "url": "https://github.com/vercel/turborepo/issues"
+  },
+  "bin": "dist/cli.js",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsup",
+    "dev": "tsup --watch",
+    "test": "jest",
+    "lint": "eslint src/**/*.ts",
+    "check-types": "tsc --noEmit"
+  },
+  "dependencies": {
+    "chalk": "2.4.2",
+    "execa": "5.1.1",
+    "fast-glob": "^3.2.12",
```

Comment:
So, fun story, npm and (pnpm/yarn) use different glob libraries. Just in case you wanted to _also_ include `minimatch` here and try to check compat. 😜